test/test/helpers/use-sinon/index.js

```diff
@@ -66,3 +66,18 @@ export function useSandbox( config, sandboxCallback = noop ) {
 		}
 	} );
 }
+
+/**
+ * Creates a sinon spy out of responseFunction and wraps it in layers of function calls, like wpcom does.
+ * Returned object stubs wpcom.
+ * responseSpy attribute of returned object provides reference to sinon spy.
+ * @param {Array} path
+ * @param {Function} responseFunction
+ * @returns {Object} wpcom-like wrapped object
+ */
+export function createWpcomSpy( path, responseFunction ) {
```

Comment:
I don't think we do.
We dont have a test in whole codebase right now that tests some arrguments in the middle.
As, in UNIT tests, we should only test the most immidiate call